source/OutfitterPanel.cpp

```diff
@@ -519,6 +527,13 @@ void OutfitterPanel::FailBuy() const
 		return;
 	}
 
+	if(!CustomSaleManager::CanBuy(*selectedOutfit))
+	{
+		GetUI()->Push(new Dialog("You can only sell this outfit here, "
+			"it is meant to be imported, generally for a good price."));
```

Comment:
Emulating the wording of the dialog above this one.
```suggestion
		GetUI()->Push(new Dialog("You can only sell this outfit here. "
			"It is being shown in the list because it is an imported item, typically "
			"sold at a higher price then normal."));
```